pkg/build/pipelines/pecl/install.yaml

```diff
@@ -0,0 +1,18 @@
+name: Installs and enables a PHP PECL module.
+
+needs:
+  packages:
+    - automake
+    - busybox
+    - cmake
```

Comment:
I'm curious about the cmake here.  Was there a PHP extension that used CMake instead of the normal PHP build system or something?

Reply:
Oh no earlier when I used make I think it was suggested to me that I should use cmake, but maybe it was just autoconf/make